Source/WebCore/html/HTMLCollection.h

```diff
@@ -101,7 +101,7 @@ class HTMLCollection : public NodeList {
 
     void invalidateNamedElementCache(Document&) const;
 
-    enum RootType { IsRootedAtNode, IsRootedAtTreeScope };
+    enum class RootType : bool { IsRootedAtNode, IsRootedAtTreeScope };
```

Comment:
Now that it's namespaced you can remove some of the redundancy here. E.g., AtNode and AtTreeScope.